client/go/outline/config/module.go

```diff
@@ -98,6 +98,17 @@ func NewDefaultTransportProvider(tcpDialer transport.StreamDialer, udpDialer tra
 	packetEndpoints.RegisterSubParser("websocket", NewWebsocketPacketEndpointSubParser(streamEndpoints.Parse))
 
 	// Stream dialers.
+	streamDialers.RegisterSubParser("iptable", func(ctx context.Context, input map[string]any) (*Dialer[transport.StreamConn], error) {
+		streamDialer, err := parseIPTableStreamDialer(ctx, input, streamDialers.Parse)
+
+		return &Dialer[transport.StreamConn]{
+			Dial: streamDialer.DialStream,
+			ConnectionProviderInfo: ConnectionProviderInfo{
```

Comment:
This needs to be fixed.
It should say tunneled only if all options are tunneled, and direct if all options are direct.
We should add a PartiallyTunneled option in case you have a mix.